### PR TITLE
TPMTI-118. Add few styles to the select component.

### PR DESCRIPTION
**Pull request by dorokhovich-mikhail**
Issue is [TPMTI-118](https://technekes.atlassian.net/browse/TPMTI-118)

### CODE REVIEW

**Comment by johnny-five** [comment]
I, @tdakhla, have checked:

### Style/Standards/Process

* [ ] If there is commented out code that should be deleted
* [ ] That the change implements what was requested in the [ticket]()
* [ ] For adequate test coverage for changed lines or critical code paths

### Code Quality

* [ ] Do the classes follow [CodeClimate 7 Patterns to Refactor](http://blog.codeclimate.com/blog/2012/10/17/7-ways-to-decompose-fat-activerecord-models/)
* [ ] If there are more idiomatic ways of using our frameworks/libraries
* [ ] Are there alternative implementations or refactors that increase readability/understandability and/or maintainability
* [ ] I can actually understand what the code is doing (if you're struggling to understand, then others may as well)
* [ ] This change is ready for production, if not should there should be a [feature flag](https://github.com/voormedia/flipflop)

### Security and Privacy

* [ ] Do the changes raise any security or privacy concerns

File: src/Select/Select.stories.tsx [hunk 1]
```diff
@@ -15,6 +15,8 @@ storiesOf('Select', module).add(
         invalid={boolean('invalid', false)}
         invalidText={text('invalidText', 'A valid value is required')}
         invalidTextColor={text('invalidTextColor', '')}
+        borderColor={text('borderColor', 'red')}
```
Comment (ben-dalton) [inline src/Select/Select.stories.tsx]: Let's default this to an empty string rather than 'red'.

File: src/Select/Select.stories.tsx [hunk 2]
```diff
@@ -15,6 +15,8 @@ storiesOf('Select', module).add(
...
+        borderRadius={text('borderRadius', '4px')}
```
Comment (ben-dalton) [inline src/Select/Select.stories.tsx]: Same here. Default to empty string so we see the default theme values when the story loads.

File: src/Select/Select.stories.tsx (hunk 1 above)
Comment (dorokhovich-mikhail) [reply src/Select/Select.stories.tsx]: Fixed

File: src/Select/Select.component.tsx [hunk 3]
```diff
@@ -72,14 +74,15 @@ const SDiv = styled.div`
   color: ${(props: Props) => props.theme.reverseText};
   .react-select-component > div {
     min-height: unset;
+    border-radius: ${(props: Props) =>
+      props.borderRadius || props.theme.common[props.selectSize!].borderRadius};
+    border-color: ${(props: Props) => props.borderColor || 'inherit'};
```
Comment (tdakhla) [inline src/Select/Select.component.tsx]: The two `border-color`s should be merged into one property.

**Comment by ben-dalton** [comment]
@dorokhovich-mikhail I added some detail to the JIRA description around what we need this component to do. Please let me know if you have any questions.

**Pull request merged**